### DISCUSSION TITLE
What is the Home Assistant app button for?

### DISCUSSION THREAD

**Discussion by mizuxe** [PRProd/HA-Firemote#332] [General]
Hi, I noticed I could select home assistant as an app button for my firemote. What's the purpose of this? As far as I know there isn't a fire tv home assistant app, so what does this button link to? I was also surprised to see that the Fire TV is the only device that this button can be used on, which left me even more confused. Is their an app I don't know about?

**Comment by PRProd** [comment]
Greetings @mizuxe!  (I really hope I'm pronouncing your name right in my head) 

I don't recall my exact thought process, but here is what I do know:

 - 24-May-2023: User @malorezzanzo64 requested a Home Assistant app shortcut in [Issue #123](https://github.com/PRProd/HA-Firemote/issues/123) and indicated that it was for Amazon Fire devices.
 - 25-May-2023: Firemote version [v2.3.2](https://github.com/PRProd/HA-Firemote/releases/tag/v2.3.2) was released and the issue was closed without any further discussion.

I took another look today, and (re)discovered that this app is not available in the Amazon app store, but it can still be sideloaded and launched using the existing Firemote button.
 - [How to install and use the Downloader app for sideloading](https://www.youtube.com/watch?v=lx7RmpD37Rs)
 - Link to Home Assistant APK to download through the Downloader app: https://home-assistant.en.uptodown.com/android/download

After doing this for myself again today, I found that I was stuck at the Home Assistant login page, and I was unable to go any further with my remote control or with any special Firemote functions.  Perhaps this is why I did not include the Home Assistant app launcher for other device families (as you likely noticed [here](https://github.com/PRProd/HA-Firemote/wiki/Firemote-App-Launchers)) but I honestly cannot remember if that was my thought process or not.

Perhaps @malorezzanzo64 can inform us about the use case for this app on the Amazon Fire platform?

Also, it's easy enough to add this button for other Android platforms as well since it can be sideloaded using the Downloader app.  If that's something that you all want, I can certainly do that.

**Reply by mizuxe** [reply]
I also couldn't get past the login page lol. Thank you for responding! Also I'm sure your pronouncing my name just fine.

**Reply by malorezzanzo64** [reply]
Hi, if I remember well I used a bluetooth keyboard to login

**Reply by mizuxe** [reply]
@malorezzanzo64 What can you do with the app on the TV? what's the benefit?  When you log in can you control using just the remote?

**Reply by malorezzanzo64** [reply]
I downloaded it to use the sensors provided by the home assistant android app. A lot of them are useless for a tv and the others maybe could be provided using adb (I don't know). The interface on the TV is really cumbersome to utilize with the remote (you can scroll through the left panel and click but nothing more)
